src/Munee/Asset/Type/Css.php

```diff
@@ -168,7 +169,8 @@ protected function isScss($file)
      */
     protected function fixRelativePaths($content, $originalFile)
     {
-        $cssParser = new CssParser($content);
+        $oSettings = CssSettings::create()->withMultibyteSupport(false);
```

Comment:
@bezumkin: Could you please rename this variable to something like `$cssParserSettings`. I don't use `o`, `s`, `i` prefixes :).  I'm going to merge this, do a patch release, and then fix it better this weekend.